loaders/src/glTF/2.0/Extensions/KHR_materials_pbrSpecularGlossiness.ts

```diff
@@ -1,59 +1,85 @@
 /// <reference path="../../../../../dist/preview release/babylon.d.ts"/>
 
 module BABYLON.GLTF2.Extensions {
+    // https://github.com/KhronosGroup/glTF/tree/master/extensions/2.0/Khronos/KHR_materials_pbrSpecularGlossiness
+
+    const NAME = "KHR_materials_pbrSpecularGlossiness";
+
     interface IKHRMaterialsPbrSpecularGlossiness {
```

Comment:
I am also wondering if this should not be part of the typings (other places might be nice to move).

Reply:
Same answer as the other. I will make a pass at some point to expose these interfaces.